packages/tooltips/src/elements/Tooltip.spec.tsx

```diff
@@ -35,47 +36,59 @@ describe('Tooltip', () => {
   });
 
   it('renders tooltip with RTL styling if provided', () => {
-    const { getByTestId } = renderRtl(<BasicExample />);
+    const { getByTestId, getByText } = renderRtl(<BasicExample />);
+
+    expect(getByText('Trigger')).not.toHaveFocus();
 
     act(() => {
-      fireEvent.focus(getByTestId('trigger'));
+      userEvent.tab();
       jest.runOnlyPendingTimers();
     });
 
+    expect(getByText('Trigger')).toHaveFocus();
     expect(getByTestId('tooltip')).toHaveStyleRule('direction', 'rtl');
   });
 
   describe('Focusable Tooltip Content', () => {
     it('remains visible if tooltip content is focused', () => {
```

Comment:
I've rewritten this test to better test real-world scenarios. It's uncommon for tooltip to contain text and be focusable. A more common scenario is that a tooltip has focusable content, say a button, as demonstrated [here](https://zendeskgarden.github.io/react-containers/storybook/?path=/story/tooltip-container--focusable-tooltip).

This test is modified so that it asserts against a more common scenario.